File: Assets/move.js
#pragma strict

var speed:float = 0.05;
function Start () {

}

function Update(){
	GetComponent.<Rigidbody2D>().velocity = Vector2(0,0);
}

function FixedUpdate () {
	
	if(Input.GetKey("w")){
		//transform.Translate(Vector2.up * speed * Time.deltaTime);
		GetComponent.<Rigidbody2D>().MovePosition(GetComponent.<Rigidbody2D>().position + Vector2.up * speed);
	}
	
	if(Input.GetKey("s")){
		//transform.Translate(Vector2.up * -speed * Time.deltaTime);
		GetComponent.<Rigidbody2D>().MovePosition(GetComponent.<Rigidbody2D>().position + Vector2.up * -speed); 
	}
	
	if(Input.GetKey("a")){
		//transform.Translate(Vector2.right * -speed * Time.deltaTime);
		GetComponent.<Rigidbody2D>().MovePosition(GetComponent.<Rigidbody2D>().position + Vector2.right * -speed); 
	}
	
	if(Input.GetKey("d")){
		//transform.Translate(Vector2.right * speed * Time.deltaTime);
		GetComponent.<Rigidbody2D>().MovePosition(GetComponent.<Rigidbody2D>().position + Vector2.right * speed); 
	}

}

File: Assets/move.cs
using UnityEngine;
using System.Collections;

public class move : MonoBehaviour {
	float moveSpeed = 3.5f;

	// Use this for initialization
	void Start () {

	}
	
	// Update is called once per frame
	void Update () {
		//Debug.Log (transform.position);

		Vector2 pos = transform.position;
		pos.x += Input.GetAxis("Horizontal") * moveSpeed * Time.deltaTime;
		transform.position = pos;

	}
}
